glue/bin/mir-kiosk-app-daemon

```diff
@@ -23,27 +23,12 @@ else
     ARCH="$SNAP_ARCH-linux-gnu"
 fi
 
-export MIR_LIBS=${SNAP}/mir-libs/$ARCH
-if [ ! -d ${MIR_LIBS} ] ; then
-    echo "####################################################################"
-    echo "ERROR: It appears ${SNAP_NAME}:mir-libs plug has not been connected."
-    echo "Please run:"
-    echo ""
-    echo "    snap install mir-libs --edge"
-    echo "    snap connect ${SNAP_NAME}:mir-libs mir-libs:mir-libs"
-    echo "    snap disable ${SNAP_NAME}"
-    echo "    sudo /usr/lib/snapd/snap-discard-ns ${SNAP_NAME}"
-    echo "    snap enable ${SNAP_NAME}"
-    echo ""
-    echo "###################################################################"
-    exit 1
-fi
-
 # XKB config
 export XKB_CONFIG_ROOT=${SNAP}/usr/share/X11/xkb
 
-# Qt Platform to Mir
-export QT_QPA_PLATFORM=ubuntumirclient
+# Qt Platform to Wayland
+export QT_QPA_PLATFORM=wayland
+export QT_QPA_PLATFORM_PLUGIN_PATH=$(find ${SNAP}/usr/lib/ -name qt5)/plugins/platforms/
```

Comment:
Is this here to determine the arch triplet? If so, you can use:

```yaml
commands:
…
    environment:
      QT_QPA_PLATFORM_PLUGIN_PATH=${SNAP}/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/plugins/platforms/
```

Reply:
You can't - $SNAPCRAFT variables are no longer available at runtime.

(Yes, this and similar code can be cleaned up, but let's do that separately.)

Reply:
They get evaluated at build time though. You can't use them in the wrapper, but you can - in `snapcraft.yaml`.

Reply:
Yes, I'm preparing a separate PR for that